src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/doclets.properties

```diff
@@ -151,8 +151,6 @@ doclet.AnnotationInterfaces=Annotation Interfaces
 doclet.Exceptions=Exceptions
 doclet.Errors=Errors
 doclet.Classes=Classes
-doclet.packages=packages
```

Comment:
I guess I'm mildly surprised to see all these items being removed...

Reply:
These were used from DeprecatedListWriter.getSummaryKey which is removed by this patch (and is unused in the current mainline, as far as I know).